packages/elements/src/pagination/index.ts

```diff
@@ -203,109 +401,54 @@ export class Pagination extends BasicElement {
+    if ((/^[1-9]([0-9]+)?$/).test(value)) {
+      return true;
     }
-
-    return 1;
-  }
-
-  /**
-   * Check a new page value to be usable
-   * if a new page value is allow then return newPage
-   * Condition to be old value is null or NaN or undefined or string or less than 1
-   * @param oldPage a old page value
-   * @param newPage a new page value
-   * @return return a new page value
-   */
-  private validatePage (oldPage: string, newPage: string): string {
-    let page = Number.parseInt(newPage, 10);
-
-    if(!page || isNaN(Number(newPage)) || isNaN(page)) {
-      page = Number.parseInt(oldPage, 10);
-    }
-    else if (page > this.totalPage) {
-      page = this.totalPage;
-    }
-    else if (page < 1) {
-      page = 1;
+    else {
+      if (value !== null && warning && propName) {
+        new WarningNotice(`${this.localName} : The specified value "${value}" of ${propName} property is not valid, The value must be integer and greater than 0.`).show();
+      }
+      return false;
     }
-
-    return page.toString();
-  }
-
-  /**
-   * Get text to display in info part
-   * @returns {PageInfo} Used for translations
-   */
-  private get pageInfo (): PageInfo {
-    const page = Number.parseInt(this.page, 10);
-    const pageSize = Number.parseInt(this.pageSize, 10);
-    const totalCount = Number.parseInt(this.totalItems, 10);
-    const from = ((page - 1) * pageSize) + 1;
-    const to = page * pageSize > totalCount ? totalCount : page * pageSize;
-
-    return {
-      from,
-      to,
-      totalCount,
-      pageSize
-    };
-  }
-
-  /**
-   * Hide info part when it's too small
-   * @returns {void}
-   */
-  private onResize (): void {
-    const display = this.clientWidth < Number.parseInt(this.getComputedVariable('--responsive-width'), 10) ? 'none' : 'block';
-    this.infoElement.style.display = display;
   }
 
   /**
-   * Handle when input is focus
-   * @returns {void}
-   */
-  private onInputFocus (): void {
-    this.input.value = this.page;
-    setTimeout(() => {
-      this.input.select();
-    });
-  }
-
-  /**
-   * Handle when input lost focus
-   * @param event Event object
-   * @returns {void}
+   * Validate number in supported range
+   * @param value value for validation
+   * @param min minimum value
+   * @param max maximum value
+   * @param warning show warning message when value is invalid
+   * @param propName property name to show in warning message
+   * @returns result of validation
    */
-  private onInputBlur (event: {target: HTMLInputElement}): void {
-    const oldPageValue = this.page;
-    this.page = this.validatePage(this.page, event.target.value);
-    // need this to update input text
-    this.requestUpdate();
-
-    if (this.page !== oldPageValue) {
-      this.notifyPropertyChange('page', this.page);
+  private validateRange (value: number, min: number, max: number, warning = false, propName = ''): boolean {
```

Comment:
Do not validate this - see comments in test files.

This can break app workflow if properties are updated in the wrong order - this can happen easily in app fraeworks like Angular. Leave dev to set correct value/max props.

Reply:
I already changed it and use it to show the warning only.